modules/installation-user-infra-exporting-common-variables-arm-templates.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-user-infra-exporting-common-variables-arm-templates_{context}"]
+= Exporting common variables for ARM templates
+
+You must export a common set of variables that are used with the provided Azure
+Resource Manager (ARM) templates used to assist in completing a user-provided
+infrastructure install on Microsoft Azure.
+
+[NOTE]
+====
+Specific ARM templates can also require additional exported variables, which are
+detailed in their related procedures.
+====
+
+.Prerequisites
+
+* Obtain the {product-title} installation program and the pull secret for your cluster.
+* Generate the Ignition config files for your cluster.
+* Install the `yq` package.
```

Comment:
I don't believe it's is a RHE package (from a standard channel).

Reply:
@fabianofranz is there a standard RHEL package we can provide instead of `yq`?

Reply:
@codyhoag not that I know, but since `yq` is only used to _read_ YAML attributes, it could be replaced with manual instructions, something like: "export the value of the `.metadata.name` attribute from the install-config.yaml file as the environment variable `CLUSTER_NAME`". There are 7 places where we read attributes from YAML files.

Reply:
@fabianofranz got it; I'll instruct the manual steps for those then.

Reply:
Updated!